Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.cpp

```diff
@@ -300,11 +300,9 @@ class LinearScan {
             }
         }
 
-        std::sort(
-            m_clobbers.begin(), m_clobbers.end(),
-            [] (Clobber& a, Clobber& b) -> bool {
-                return a.index < b.index;
-            });
+        std::ranges::sort(m_clobbers, [] (Clobber& a, Clobber& b) -> bool {
```

Comment:
Can remove the space, use auto, and remove the -> bool.

```suggestion
        std::ranges::sort(m_clobbers, [](auto& a, auto& b) {
```